INSTALL.md

```diff
@@ -1,242 +1,190 @@
 # Installation and Configuration for cfgov-refresh
 
 There are two ways to install:
- - [Vagrant-box installation](INSTALL.md#vagrant-box-installation)
- - [Stand-alone installation](INSTALL.md#stand-alone-installation)
+- [Stand-alone installation](#stand-alone-installation)
+- [Vagrant-box installation](#vagrant-box-installation)
+  :warning: These instructions are not currently working.
```

Comment:
@Scotchester @willbarton   FYI, the vagrant instructions are working.   

The vagrant box doesn't load all the optional applications, however.